doc/source/examples/examples.rst

```diff
@@ -10,3 +10,4 @@ Examples
     incompressible-flow/incompressible-flow
     multiphysics/multiphysics
     dem/dem
+    resolved-cfd-dem/resolved-cfd-dem
```

Comment:
Toni created a cfd-dem section, I don't know if we want to separate them this way or have them all together in the cfd-dem folder.

Reply:
For now, I will let them separate as they are separated in the parameters section as well.